scripts/check_enterprise_imports.sh

```diff
@@ -9,7 +9,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
 cdroot
 
 set +e
-find . -regex ".*\.go" | grep -v "./enterprise" | xargs grep -n "github.com/coder/coder/enterprise"
+find . -regex ".*\.go" | grep -v "./enterprise" | grep -v "./scripts/auditdocgen/main.go" | xargs grep -n "github.com/coder/coder/enterprise"
```

Comment:
`grep -v "./scripts/auditdocgen/*.go"` - in case we add more .go files in the future.

Reply:
Good idea! I'll add this change in [this PR](https://github.com/coder/coder/pull/5887).